src/components/icon/icon.tsx

```diff
@@ -159,7 +164,7 @@ export class Icon {
           [mode]: true,
           ...createColorClasses(this.color),
           [`icon-${this.size}`]: !!this.size,
-          'flip-rtl': !!flipRtl && isRTL(el),
+          'flip-rtl': Boolean(this.shouldBeFlippable),
```

Comment:
```suggestion
          'flip-rtl': shouldBeFlippable,
```

If we avoid the optional chaining then we likely don't need to coerce this to a boolean.